cudapoa/tests/Test_CudapoaNW.cpp

```diff
@@ -197,26 +197,28 @@ NWAnswer testNW(const BasicNW& obj)
     int16_t mismatch_score;
     int16_t match_score;
     uint16_t* aligned_nodes; //local; to store num of nodes aligned (length of alignment_graph and alignment_read)
+    UpperLimits max_limits;
+    max_limits.setLimits(1024);
```

Comment:
do we need this here if the default is `1024`?

Reply:
you're right, considering that the default is 1024, it's not needed. At the time of adding this line, I thought just in case the default value changes later, this will ensure the Test file behaves as before. Will remove and add a comment.